### DISCUSSION TITLE
Make zooming in faster

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#3578] [Feedback]
Maybe it is just me but I am repeatedly a bit irritated by waiting for zooming in/out on element selection and quest solving. Especially where there multiple elements that I solve together.

Maybe increasing animation speed a bit would be a good idea?

**Comment by smichel17** [comment]
Maybe a little? Hard for me to comment since I have all my animations set to half duration.

On December 9, 2021 11:06:55 AM UTC, Mateusz Konieczny ***@***.***> wrote:
>Maybe it is just me but I am repeatedly a bit irritated by waiting for zooming in/out on element selection and quest solving. Especially where there multiple elements that I solve together.
>
>Maybe increasing animation speed a bit would be a good idea?
>

-- 
To respect your time, I try to write short, functional emails.

**Reply by mnalis** [reply]
How did you do it? Is it some general Android setting, or you have your custom build modified to be faster?

**Reply by smichel17** [reply]
General android settings > Developer Options > Animator duration scale = 0.5x

Didn't you have animations disabled this way at one point?

Note: It doesn't affect *all* animations: the quest form still slides onto the screen, even if Window animation, Transition animation, and Animator duration scale are set to "off".

**Reply by mnalis** [reply]
I don't think it was me (although my memory could be failing. I did try enabling _audio effects_ in SC and then quickly turned it back off, perhaps that ringed the bell? Or the whole _"wrong Notes position with pictures" bug_ where TangramES animations did jump all over the place?)

Anyway trying that `Animator duration scale = 0.5x` is just about too fast for me (and with animations off, SC  looks much worse usability-wise for me). Unfortunately those are the only choices for speeding up (I guess `0.75x` as a halfway between `0.5x` and default `1x` would be about fine "fast" speed for me - but Android does not offer that). 

But perhaps mentioning that Android option in  [FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) might be good for @matkoniecz and others for whom SC animations are too slow (or too fast)?

**Comment by mnalis** [comment]
Maybe?

However I find initial zoom-in animation (right after clicking on quest) useful to help verify if the right quest is selected / for orientation, and I'm not sure what speed would be too fast for me (or other users) to follow. Slight increase should be OK if you ask me; but then again, slight increase won't really help much with the issue...

Zoom-out (after the quest is solved) IMHO could be made much faster, as it is less useful. Not sure if different speeds for zoom in/out would be annoying, though.

And I guess _"make it an user preference"_ would be (appropriately) frowned at :smiley:
